src/Components/Input/input.scss

```diff
@@ -0,0 +1,15 @@
+input {
+  display: flex;
+  height: 30px;
+  border-radius: 20px;
+
+  &::placeholder {
+    padding-left: 10px;
+    color: gray;
+  }
+
+  &:focus-visible {
+    outline: none !important;

```

Comment:
do you need the `!important` to get it working? I mean, it's always better to not use `!important`